go.mod

```diff
@@ -131,3 +131,6 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+// Fixes CVE-2024-45338
+replace golang.org/x/net => golang.org/x/net v0.33.0
```

Comment:
Just curious, can't we bump the whole `golang.org/x/net` to v0.33.0 rather than being on v0.30.0 and having to do the replace?

If not maybe we can investigate which other dependency is enforcing the lower version and try to bump that.

Reply:
changed as suggested.